### PR TITLE
feat(external-node): add helm chart

### LINKED ISSUE #2: devops(external-node): easy replication
# Purpose

- Provide template to create external-nodes on-demand
- Check if creating extra postgres instances with gke may help: Deploy a highly-available PostgreSQL database on GKE  |  Kubernetes Engine  |  Google Cloud
- Investigate kustomize, helm, jinja or any other tool that facilitates this job. Terraform has to be scalable too, this may be the most difficult part.
  - generate IP per external-node?
  - access external-nodes via an ingress?

### PR DESCRIPTION
**Pull request by fborello-lambda**
Tested in a previous stage. It has to be fully tested whenever we have the database for the external nodes up and running, and the snapshots created.
closes #2

### CODE REVIEW

File: infra/helm-charts/external-node/templates/defaultbackend-ingress-grafana.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        app: default-backend-ingress-grafana
+
+    spec:
+      containers:
+      - name: default-backend-ingress-grafana
+        image: nginx:latest
+        resources:
+          limits:
+            memory: "128Mi"
+            cpu: "500m"
```
Comment (ManuelBilbao) [inline infra/helm-charts/external-node/templates/defaultbackend-ingress-grafana.yaml:38]: Why is this needed?

**Review by ManuelBilbao — DISMISSED** [review]
LGTM

File: infra/helm-charts/external-node/templates/defaultbackend-ingress-grafana.yaml (hunk 1 above)
Comment (fborello-lambda) [reply infra/helm-charts/external-node/templates/defaultbackend-ingress-grafana.yaml:38]: It's not strictly necessary, but it serves a simple webpage with an error message. For now it can be treated as a template, we can delete the service and add it later on if we need it.
We can use this error webpage to include links to each Grafana service domain.

Comment (fborello-lambda) [reply infra/helm-charts/external-node/templates/defaultbackend-ingress-grafana.yaml:38]: And if we are talking just about the `limits` we can delete them.

**Review by klaus993 — DISMISSED** [review]

**Comment by klaus993** [comment]
LGTM
You need to do GPG/SSH signing set-up, amend your commit and force-push it.
Info on how to do that with GPG:
```
brew install gpg
```
and then follow these two docs:
* https://docs.github.com/en/authentication/managing-commit-signature-verification/generating-a-new-gpg-key
* https://docs.github.com/en/authentication/managing-commit-signature-verification/telling-git-about-your-signing-key

**Review by klaus993 — APPROVED** [review]
